canonical-web-and-design/react-components
# Remove custom button styles

## Done

- Remove custom styling from ActionButton
- Add `small` button introduced in Vanilla 2.20.0
- Drive-by: remove random data from ModularTable example to prevent Percy from flagging difference every time

Fixes #337

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Make sure `ActionButton` works as before
- Check `Button` and new `small` prop that should enable small button
